sql/create-db.sql

```diff
@@ -37,6 +37,7 @@ CREATE TABLE IF NOT EXISTS instances (
                                     -- to ensure that we have correct ordering of cloud watch
                                     -- events
   touched TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  hasClaimedCredentials BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
please ensure that the `lib/state.js` methods for instances allow for creating instances with this value set explicitly to true or false.